src/OpenLoco/src/Ui/Windows/CompanyWindow.cpp

```diff
@@ -200,11 +202,11 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
 
             if (CompanyId(self.number) == CompanyManager::getControllingId())
             {
-                self.widgets[widx::change_owner_name].type = WidgetType::buttonWithImage;
+                self.widgets[widx::change_owner_name].hidden = false;
```

Comment:
I get that this was probably a find/replace job, but I'd now rather we _use_ the boolean expression instead of branching here, e.g.:

`self.widgets[widx::change_owner_name].hidden = CompanyId(self.number) != CompanyManager::getControllingId();`